Hivemind/Assets/Scripts/ResourceNode.cs

```diff
@@ -29,12 +29,23 @@ private void Awake()
         resourceAmount = BaseResourceAmount;
         futureResourceAmount = resourceAmount;
         myResourceNode = gameObject;
-        resourceNodeTransform = gameObject.transform; GameWorld.AddNewResource(this);
+        resourceNodeTransform = gameObject.transform;
+        myResourceNode.GetComponent<MeshRenderer>().enabled = false;
+        GameWorld.AddNewResource(this);
+    }
+
+    public void AddToKnownResourceList()
+    {
+        if (!GameWorld.KnownResources.Contains(this) && myResourceNode != null)
+        {
+            myResourceNode.GetComponent<MeshRenderer>().enabled = true;
+            GameWorld.AddNewKnownResource(this);
+        }
     }
 
     private void Update()
     {
-        if (resourceAmount < BaseResourceAmount && !respawningResources)
+        if (resourceAmount < BaseResourceAmount && !respawningResources && resourceType != ResourceType.Crystal)
```

Comment:
hoezo kan crystal nooit respawnen? dat is toch gewoon een vinkje binnen de crystal prefab en of instanties daarvan?
als je dit veranderd naar dat vinkje zou je ook grabResource aan moeten passen

Reply:
Ik heb nu een cherckbox gezet voor of een resource can respawnen en of een resource vernietigt moet worden wanneer deze leeg is